src/chains/ethereum/ethereum/tests/api/eth/getTransactionReceipt.test.ts

```diff
@@ -125,26 +135,26 @@ describe("api", () => {
           );
         });
 
-        it("doesn't log if legacyInstamine is enabled", async () => {
-          const legacyInstamineProvider = await getProvider({
+        it("doesn't log if instamine is set to 'eager' (default)", async () => {
+          const eagerInstamineProvider = await getProvider({
             logging: { logger },
-            miner: { legacyInstamine: true }
+            miner: { instamine: "eager" }
           });
 
-          const [from] = await legacyInstamineProvider.send("eth_accounts");
+          const [from] = await eagerInstamineProvider.send("eth_accounts");
 
-          const hash = await legacyInstamineProvider.send(
+          const hash = await eagerInstamineProvider.send(
             "eth_sendTransaction",
             [{ from, to: from }]
           );
 
-          const result = await legacyInstamineProvider.send(
+          const result = await eagerInstamineProvider.send(
             "eth_getTransactionReceipt",
             [hash]
           );
 
           // the tx is mined before sending the tx hash back to the user
-          // if legacyInstamine is enabled - so they will get a receipt
+          // if eagerInstamine is enabled - so they will get a receipt
```

Comment:
```suggestion
          // if instamine is set to 'eager' - so they will get a receipt
```
Nit-picking, but changing to be consistent.